nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -374,6 +374,8 @@ in
                 '')}
               '';
 
+            # restart sshd on "/etc/ssh/sshd_config" content change. this does not abort established connections
+            restartTriggers = [ (builtins.hashString "sha256" config.environment.etc."ssh/sshd_config".text) ];
```

Comment:
Is there a particular reason to explicitly compute a hash of the text here?
Why not  `restartTriggers = [ config.environment.etc."ssh/sshd_config".source ];` as in other modules?

Reply:
Like this one? https://github.com/NixOS/nixpkgs/pull/53050